decidim-templates/app/views/decidim/templates/admin/questionnaire_templates/edit.html.erb

```diff
@@ -1,3 +1,4 @@
+<% add_decidim_page_title(t(".title")) %>
 <% add_decidim_page_title(t(".questionnaire")) %>
```

Comment:
This isn't visible in the header title ("item_show__header-title"), so let's leave it out 

```suggestion
```